crates/bevy_core_widgets/src/core_button.rs

```diff
@@ -24,7 +26,7 @@ pub struct CoreButton {
     /// Optional system to run when the button is clicked, or when the Enter or Space key
     /// is pressed while the button is focused. If this field is `None`, the button will
```

Comment:
Comments still mention `None`, and also does this still emit an Event in some way, perhaps when its `Ignore`?

Reply:
I had meant to delete that part of the comment (in fact, I did so in another branch).

Done.